core/project_settings.cpp

```diff
@@ -588,6 +597,10 @@ Error ProjectSettings::save() {
 	return save_custom(get_resource_path().plus_file("project.godot"));
 }
 
+bool ProjectSettings::is_settings_text_up_to_date() {
```

Comment:
Seems like the non-custom version is never used.

Reply:
Yeah, I added it for consistency with the save() function. Should I remove it ?